### PR TITLE
Add a JUnit report

### PR DESCRIPTION
**Pull request by marob**

### CODE REVIEW

**Comment by tisto** [comment]
@addyosmani any reason why this pull request hasn't been reviewed/merged yet? Is there anything I could do to help to make this happen?

File: index.js [hunk 1]
```diff
@@ -31,6 +32,8 @@ module.exports = function (url, opts, cb) {
         height: viewportSize[1] || 768
     });
 
+    var startTimestamp = new Date().getTime();
```
Comment (sindresorhus) [inline index.js]: Should use `Date.now()`

File: index.js [hunk 2]
```diff
@@ -43,6 +46,12 @@ module.exports = function (url, opts, cb) {
             return;
         }
 
-        cb(null, parseJson(stdout));
+        var endTimestamp = new Date().getTime();
```
Comment (sindresorhus) [inline index.js]: Should use `Date.now()`

**Comment by sindresorhus** [comment]
I would recommend just creating an external package and feed it the result of running `a11y`. No reason it has to be part of this package. One missing thing in the `result` is `duration`, but we can add.

**Comment by addyosmani** [comment]
Pardon the delay. I've been trying to get out a release of the upstream accessibility developer tools package (finally out this week!). I agree with Sindre on this being better outside of this package.

**Comment by marob** [comment]
@sindresorhus the "url" would be missing too.
I've developed this feature in the core module (a11y) to be able to use it from using a11y directly, but also grunt-a11y or gulp-a11y without having to re-develop everything for each wrapper.

How would you move this to an external package while enabling it to be used from a11y or any wrapper?

**Comment by sindresorhus** [comment]
@marob You already have the `url`, since it's coming from you, but we could add that too if it's more convenient.

> How would you move this to an external package while enabling it to be used from a11y or any wrapper?

The wrappers add your reporter as a dependency. They would have had to document the option and do the right thing regardless.

**Comment by marob** [comment]
FYI, I managed to create a JUnit reporter external package: https://github.com/Smile-SA/a11y-junit-reporter
And I integrated it in grunt-a11y: https://github.com/lucalanca/grunt-a11y/pull/18

**Pull request closed without merging**